clusterMembership/staticCluster.go

```diff
@@ -11,46 +8,19 @@ import "errors"
  * Keeps a Map of the nodeId to Node for fast lookup of nodes.
  */
 type staticCluster struct {
-	members    []string
-	membersMap map[string]Node
+	members []Node
 }
 
-func (c staticCluster) Members() []string {
+func (c staticCluster) Members() []Node {
 	return c.members
```

Comment:
Same, I was trying to avoid it but then having the Cluster Manager own sending work to nodes seemed broken.  This basically relies on the fact that users won't mutate things, and this should be treated as readonly.